### PR TITLE
Improve resolution for Slandles

### PR DESCRIPTION
**Pull request by Cypher1**

### CODE REVIEW

File: src/runtime/test/manifest-test.ts [hunk 1]
```diff
@@ -644,45 +644,46 @@ ${particleStr1}
...
+      assert.equal(recipe.isResolved(options), expectedIsResolved, `${label}: Expected recipe to be ${expectedIsResolved ? '' : 'un'}resolved.\nErrors: ${JSON.stringify([...options.errors, options.details])}`);
     };
-    await parseRecipe({isRequiredSlotA: false, isRequiredSlotB: false, expectedIsResolved: true});
-    await parseRecipe({isRequiredSlotA: true, isRequiredSlotB: false, expectedIsResolved: true});
-    await parseRecipe({isRequiredSlotA: false, isRequiredSlotB: true, expectedIsResolved: false});
-    await parseRecipe({isRequiredSlotA: true, isRequiredSlotB: true, expectedIsResolved: false});
+    await parseRecipe('1', false, false, true);
+    await parseRecipe('2', true, false, true);
+    await parseRecipe('3', false, true, false);
+    await parseRecipe('4', true, true, false);
```
Comment (shans) [inline src/runtime/test/manifest-test.ts]: I think the idea of the args object was that it was documenting the meaning of the boolean flags.

File: particles/canonical.manifest [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
+import 'Profile/SLANDLESProfile.recipes'
+import 'Products/SLANDLESProducts.recipe'
+import 'Events/SLANDLESEvents.recipes'
+import 'Restaurants/SLANDLESRestaurants.recipes'
+import 'Music/SLANDLESMusic.recipes'
+import 'TVMaze/SLANDLESTVMaze.recipes'
+import 'Words/SLANDLESWords.recipes'
+import 'Services/SLANDLESServices.recipes'
+import 'Stardate/SLANDLESStardate.recipes'
+import 'Demo/SLANDLESDemo.recipes'
```
[the remark this answers is not in the document]
Comment (Cypher1) [reply particles/canonical.manifest]: Reset these: they were only for testing.

File: src/runtime/test/manifest-test.ts (hunk 1 above)
Comment (Cypher1) [reply src/runtime/test/manifest-test.ts]: I think things have improved thanks to our use of typescript.
I'll put it back for now (and use it in the non-slandles test as well) until I feel more strongly about it, as I'm currently unsure that the conciseness is worth the potential cost to readability .

Comment (shans) [reply src/runtime/test/manifest-test.ts]: Interesting point re: typescript. Maybe it's worth following up with Maria (she wrote these tests)?

**Pull request merged**